### DISCUSSION TITLE
Immediately losing adb commands upon disconnect

### DISCUSSION THREAD

**Discussion by CaptainFrosty** [RikkaApps/Shizuku#202] [Q&A]
Hi, my phone is not rooted and used to keep adb until reboot, but lately I lose it when I disconnect from adb. I have not changed the USB hardware I'm using.

This affects other things as well, so I am posting here.

**Comment by tenclr** [comment]
I also had the same problem as you, always losing connections in the night.

**Comment by linuxct** [comment]
Have you tried starting `shizuku_starter` with `nohup`?
As long as you don't mess with your ADB configuration in your device, it should continue to work, regardless of whether it is plugged to the computer or not.
